src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java

```diff
@@ -204,11 +206,28 @@ private void videoCriteriaConvertPanel() {
         gbc_checkBoxKeyFrame.gridy = 13;
         add(checkBoxKeyFrame, gbc_checkBoxKeyFrame);
 
+        JLabel lblFormatKeyframe = new JLabel("Format:");
+        GridBagConstraints gbc_lblFormatKeyframe = new GridBagConstraints();
+        gbc_lblFormatKeyframe.anchor = GridBagConstraints.EAST;
+        gbc_lblFormatKeyframe.insets = new Insets(0, 0, 5, 5);
+        gbc_lblFormatKeyframe.gridx = 2;
+        gbc_lblFormatKeyframe.gridy = 13;
+        add(lblFormatKeyframe, gbc_lblFormatKeyframe);
+
+        comboBoxKeyFrameFormat = new JComboBox();
+        comboBoxKeyFrameFormat.setModel(new DefaultComboBoxModel(new String[]{"png", "jpg", "tiff"}));
```

Comment:
All previous implemented comboBox show options in uppercase.

Reply:
Solved.